docs/getting-started.md

```diff
@@ -2,6 +2,8 @@
 
 ## Installing Eno
 
+At the begining, install the Eno CRD resource defined at [here](https://github.com/Azure/eno/tree/main/api/v1/config/crd) to your cluster.
```

Comment:
Great catch. I'm thinking we could simplify onboarding a bit by concatenating the deploy.yaml manifest with the generated CRDs into a release artifact like `manifest.yaml`. Then swap out the file we curl in the steps below for that one